### PR TITLE
Update LIMITATIONS_AND_TROUBLESHOOTING.md

### PR DESCRIPTION
**Pull request by snoyer**
small typo

### CODE REVIEW

**Review by mwestphal — APPROVED** [review]

File: doc/user/LIMITATIONS_AND_TROUBLESHOOTING.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ F3D is not signed by Microsoft yet so Windows shows this warning. Just click on
 
 > Using the binary release, I'm unable to run F3D because macOS warns about F3D not being signed.
 
-F3D is not signed by Apple yes so macOS shows this warning. To run F3D, right click on the app and click "open", then click "open" again to be able to run F3D.
+F3D is not signed by Apple yet so macOS shows this warning. To run F3D, right click on the app and click "open", then click "open" again to be able to run F3D.
 
 
 ### Thumbnails
```
Comment (snoyer) [inline doc/user/LIMITATIONS_AND_TROUBLESHOOTING.md:52]: also not sure what's up with the header level here?

Comment (mwestphal) [reply doc/user/LIMITATIONS_AND_TROUBLESHOOTING.md:52]: right, it is wrong.

Comment (mwestphal) [reply doc/user/LIMITATIONS_AND_TROUBLESHOOTING.md:52]: Could you fix it up ? Windws/Linux/MacOS should be ##

Comment (snoyer) [reply doc/user/LIMITATIONS_AND_TROUBLESHOOTING.md:52]: are the "Thumbnails" and "Sandboxing" sections supposed to be `h3` under the "MacOS" `h2`? seems like they're about linux stuff

**Comment by snoyer** [comment]
Best-guess at sorting the subsections.
I removed the level 3 headers to match the rest of the document but maybe every "> issue" needs a header to make it clearer?

Also should I run the formatter on the markdown?

File: doc/user/LIMITATIONS_AND_TROUBLESHOOTING.md (hunk 1 above)
Comment (mwestphal) [reply doc/user/LIMITATIONS_AND_TROUBLESHOOTING.md]: yup, that linux stuff, I made a mistake yesterday.,

**Comment by mwestphal** [comment]
its good now.

**Review by mwestphal — APPROVED** [review]

**Pull request merged**